### PR TITLE
[6CORE-003] created skeleton of Order Management service

### PR DESCRIPTION
**Pull request by kateryna-mykh**

### CODE REVIEW

File: 3-platform/order-ms/app/pom.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>app</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.6core.platform</groupId>
+            <artifactId>domain</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
```
Comment (sokimaaa) [inline 3-platform/order-ms/app/pom.xml]: bad name for groupId and artifactId. 
We have a bunch of services, and with such naming we have a lot of overlapping dependency (you are expecting here is order-related domain, someone else will expect the warehouse related domain).
Group Id describes company name, so you can left as it is. 
Change Artifact Id. 
```suggestion
            <groupId>com.6core.platform</groupId>
            <artifactId>domain</artifactId>
            <version>0.0.1-SNAPSHOT</version>
```

File: 3-platform/order-ms/app/src/main/java/com/core/platform/orderms/app/OrderUseCase.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package com.core.platform.orderms.app;
+
+import com.core.platform.orderms.domain.model.OrderRecord;
+import com.core.platform.orderms.domain.persistent.OrderRepository;
+import com.core.platform.orderms.domain.service.OrderService;
+
+public final class OrderUseCase {
```
Comment (sokimaaa) [inline 3-platform/order-ms/app/src/main/java/com/core/platform/orderms/app/OrderUseCase.java]: we don't need these dummy classes, leave packages empty, or add package-info.java to reflect package structure in git.

File: 3-platform/order-ms/app/src/main/java/com/core/platform/orderms/app/OrderUseCase.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package com.core.platform.orderms.app;
```
Comment (sokimaaa) [inline 3-platform/order-ms/app/src/main/java/com/core/platform/orderms/app/OrderUseCase.java]: bad naming of package -> it should be sum of groupId + artifactId. 
In java package names is reversed name of company, e.g. for mate.academy.com -> package name will `com.mate.academy`.
```suggestion
package com._6core.platform.orderms.app;
```

File: 3-platform/order-ms/pom.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <modules>
+        <module>boot</module>
+        <module>infra</module>
+        <module>app</module>
+        <module>domain</module>
+    </modules>
+</project>
```
Comment (sokimaaa) [inline 3-platform/order-ms/pom.xml]: remove all redundant classes. 
They were just examples of how it may look.

**Review by sokimaaa — CHANGES_REQUESTED** [review]

**Review by sokimaaa — APPROVED** [review]
You still need the approve from at least one team member.

**Review by VictoriaKh-T — APPROVED** [review]
Ok

**Pull request merged**